CMakeLists.txt

```diff
@@ -10,6 +10,9 @@ endif()
 if(MSVC)
 	add_definitions(-D_CRT_SECURE_NO_WARNINGS)
 	add_definitions(-D_CRT_SECURE_NO_DEPRECATE)
+    set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} /W4 /WX")
+    set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} /wd4324") # structure was padded due to alignment specifier
```

Comment:
I'm no fan of this as dl is said to "compile out of the box". I would rather see these silenced in code so to not force this upon the user!

Reply:
Okay. It might be that DL compile out of the box with W3 today, but not with W4?
I added these since I got autobuilder failures which I didn't see locally when developing with cmake. But I don't know which compiler switches the autobuilder uses?

Reply:
https://github.com/wc-duck/datalibrary/blob/master/bam.lua#L181 I see now that I had these silenced myself... I'll see myself out and open a new issue to fix these correctly :)